addons/mail/static/src/core/public_web/thread_model_patch.js

```diff
@@ -31,9 +31,8 @@ patch(Thread.prototype, {
         ) {
             if (this.model === "discuss.channel" && this.inChathubOnNewMessage) {
                 await this.store.chatHub.initPromise;
-                let chatWindow = this.store.ChatWindow.get({ thread: this });
-                if (!chatWindow) {
-                    chatWindow = this.store.ChatWindow.insert({ thread: this });
+                if (!this.channel.chatWindow) {
+                    const chatWindow = this.store.ChatWindow.insert({ channel: this.channel });
```

Comment:
All of the `insert({ channel` with dict could be `insert(channel)` with single id).